Examples/Swift/InsetMapExample.swift

```diff
@@ -0,0 +1,61 @@
+import Mapbox
+
+@objc(InsetMapExample_Swift)
+
+class InsetMapExample_Swift: UIViewController, MGLMapViewDelegate {
+    var mapView: MGLMapView!
+    var miniMapview: MGLMapView!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        mapView = MGLMapView(frame: view.bounds)
+        mapView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        mapView.delegate = self
+        
+        // Set the main map's center coordinate and zoom level.
+        mapView.setCenter(CLLocationCoordinate2D(latitude: 18.1096, longitude: -77.2975), zoomLevel: 9, animated: false)
+
+        // Set inset mapView's center
```

Comment:
Please refer to the map as a `map view` instead of `mapView` when referring to `MGLMapView` in this context.